LLama/LLamaContext.cs

```diff
@@ -564,6 +564,35 @@ public Task<DecodeResult> DecodeAsync(LLamaBatchEmbeddings batch, CancellationTo
         {
             return Task.Run(() => Decode(batch), cancellationToken);
         }
+
+        /// <summary>
+        /// </summary>
+        /// <param name="tokens"></param>
+        /// <param name="id"></param>
+        /// <param name="batch"></param>
+        /// <param name="n_past"></param>
+        /// <returns></returns>
+        public (DecodeResult, int) Decode(List<LLamaToken> tokens, LLamaSeqId id, LLamaBatch batch, ref int n_past)
```

Comment:
Is this method needed anywhere by your other changes? I don't see anywhere it's called.

Reply:
No, it's added only for the API to be symmetric and can be removed.

Reply:
Normally I'd say to keep it for the sake of API completeness. However, in this case the `n_past` style of calling `Decode` is legacy, and will be going away once we manage to replace the Executors.

Reply:
Fixed

Reply:
Thanks